### PR TITLE
 feat(decision-table): remove decision id cell

### LINKED ISSUE #496: Remove decision id component from decision table
__What should we do?__

Remove the ability to show and edit the decision id in a decision table.

__Why should we do it?__

We decided to not have this one inside the new decision table layout

image

Child of #494

### PR DESCRIPTION
**Pull request by barmac**
Closes #496

### CODE REVIEW

**Comment by barmac** [comment]
I guess so. I wasn't aware of that.

**Comment by pinussilvestrus** [comment]
I also wonder a bit why this test case is not failing as well...

https://github.com/bpmn-io/dmn-js/blob/2e57786d46da2fda94e605733f14e66115723308/packages/dmn-js-decision-table/test/spec/features/keyboard/KeyboardSpec.js#L146

**Comment by barmac** [comment]
I'm going to release `diagram-js` so that we can actually see the test results.

**Comment by barmac** [comment]
Fun fact: test cases start to fail when you run only `dmn-js-decision-table` tests.

**Comment by barmac** [comment]
I am going to hold it until we resolve the unrelated tests breaking problem.

File: packages/dmn-js-decision-table/src/features/keyboard/Keyboard.js [hunk 1]
```diff
@@ -206,7 +206,7 @@ export default class Keyboard {
       const selectedCell = cellSelection.getCellSelection();
 
       // add new rule if no next rule
-      if (!changed && selectedCell && !isDecisionTableIdCell(selectedCell)) {
+      if (!changed && selectedCell) {
```
Comment (pinussilvestrus) [inline packages/dmn-js-decision-table/src/features/keyboard/Keyboard.js:209]: By removing this one we have to ensure the "add new rule on enter" action got not triggered on the name property input

Kapture 2020-04-27 at 9 14 32

Comment (barmac) [reply packages/dmn-js-decision-table/src/features/keyboard/Keyboard.js:209]: Oh 😱 . I'm gonna fix that. Thanks!

**Comment by philippfromme** [comment]
What's the status here?

**Comment by barmac** [comment]
The status is I need to fix https://github.com/bpmn-io/dmn-js/pull/502#discussion_r415567197, but I first need to finish the Zeebe Modeler stuff.

**Comment by philippfromme** [comment]
👍

File: packages/dmn-js-decision-table/src/features/keyboard/Keyboard.js (hunk 1 above)
Comment (barmac) [reply packages/dmn-js-decision-table/src/features/keyboard/Keyboard.js]: Fixed that.

**Pull request merged**